### ISSUE TITLE
Septic pulmonary emboli

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6540]
**Preferred term label:** Septic pulmonary emboli

**Synonyms**

**Definition (free text, please give PubMed ID)**
Embolisation of intravascular thrombus containing microorganisms into the lung via pulmonary arterial system. 

**Parent term (use hpo.jax.org/app)**
Septic emboli (to be created)

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik  -- we should probably add a few other septic <organ> embolism term. What do you think about the list here
https://www.ncbi.nlm.nih.gov/books/NBK549827/

**Comment by eswietlik** [comment]
@pnrobinson  Great idea. Shall I add them?

**Comment by pnrobinson** [comment]
@eswietlik  is this definition ok?

Embolization of intravascular thrombus containing microorganisms into the pulmonary parenchyma via arterial system. Septic pulmonary embolism can be associated with multiple additional clinical manifestations such as fever, tachypnea, and hemoptysis. This HPO term refers to the finding of the septic embolus in the lung, which can be inferred from radiological findings. Typical radiographic features of septic PE include patchy air space lesions simulating non-specific bronchopneumonia; multiple ill defined round or wedge shaped densities of varying sizes from 0.5 to 3.5 cm located peripherally; lesions abutting the pleura and located at the end of vessels (feeding vessel sign) seen on chest CT scans. Other pulmonary features suggesting septic PE include bilateral, occasional unilateral, rapid progression of cavities or abscess formations.

https://pubmed.ncbi.nlm.nih.gov/12244005/

**Comment by pnrobinson** [comment]
added
